doc/source/methods/TreeSHAP.ipynb

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
<p>the expected values is computed --&gt; the expected values are computed</p><p>note --&gt; node</p><p>ensamble --&gt; ensemble</p><p>ensamles --&gt; ensembles</p><p>hessian --&gt; Hessian</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>